docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
The big downside to installing it in zulip-npm-cache is that yarn won't be in the default location expected. We will need to simlink it to /usr/bin or it won't be available on the path. Which might be ok it we assume that users shouldn't call yarn directly.